src/index.js

```diff
@@ -60,12 +61,27 @@ export function inject(container, options) {
   const workspace = Blockly.inject(container, options);
   workspace.getRenderer().getConstants().selectedGlowFilterId = '';
 
+  // Add a drop shadow to mid-drag blocks.
+  workspace.addChangeListener((event) => {
+    if (event.type === Blockly.Events.BLOCK_DRAG) {
+      if (event.isStart) {
+        event.blocks[0].getSvgRoot().setAttribute(
```

Comment:
Can you set filters via css? If so, then can you use the `blocklyDragging` class instead?

That class does apply to workspace comments as well so you might have to check for `not(.blocklyComment)`, until we actually add a class for blocks.

Reply:
Ooh yes, and workspace comments also have drop shadows, so that's a plus actually.